lib/settings.py

```diff
@@ -1,6 +1,6 @@
 import os
 import yaml
-from lib.cli.config import config_user_name, config_email_address, config_email_host
+from lib.cli.config import config_user_name, config_email_address, config_email_host, config_model_language
```

Comment:
Does the settings module really need to be connected to the CLI? I would like to reuse this for the GUI as well.
You don't need to fix this now but something to keep in mind.

Reply:
Yeh, I agree that it should be separate from the cli. But I think that that is out of the scope for this pr, will make a new issue for it.

Reply:
Made issue #111 to solve this.